docs/how_to_knit/basics.md

```diff
@@ -1,22 +1,35 @@
 
 ## Single Bed Fair Isle
-For your first practice piece, load a test pattern such as:
+
+To create your own patterns, refer to the AYAB manual, section "pattern_image_creation". Make sure the 
+width of the pattern has an even number of pixels (=stitches).  
```

Comment:
Where does this requirement come from? The AYAB software can handle patterns with uneven width as well.

Reply:
For single bed fair isle, this requirment does not exist. I forgot to remove it - will fix it in a minute